src/main/java/dk/aau/cs/model/tapn/TAPNQuery.java

```diff
@@ -27,9 +24,11 @@ public int getExtraTokens() {
 	}
 	
 	public QueryType queryType(){
-		if(property instanceof TCTLEFNode) return QueryType.EF;
-		else if(property instanceof TCTLEGNode) return QueryType.EG;
-		else if(property instanceof TCTLAFNode) return QueryType.AF;
+		if (property instanceof TCTLEFNode) return QueryType.EF;
+		else if (property instanceof TCTLEGNode) return QueryType.EG;
+		else if (property instanceof TCTLAFNode) return QueryType.AF;
+		else if (property instanceof LTLENode) return  QueryType.E;
+		else if (property instanceof LTLANode) return QueryType.A;
 		else return QueryType.AG;
```

Comment:
This looks weird, I would suggest to add a test for AG and in "else" return unknown.

Reply:
Should be fixed now